### DISCUSSION TITLE
Fetch

### DISCUSSION THREAD

**Discussion by vinith2407** [community/community#61712] [Apps, API and Webhooks]
Git fetch upstream is not working when I give the correct password credentials

**Comment by saqer23** [comment]
The fetch did work, but to see the result, you should do a:

git log --oneline --all --graph --decorate --branches

You would see the upstream branches (including the upstream/master)

You can also check the latest SHA1 fetched with:

git branch -avv

The git pull upstream master didn't repeat the fetch part (since it was already done), but merge as well upstream/master to master.

**Comment by engrmh** [comment]
When you say "not working," can you provide more details about the issue you're experiencing? Are you getting any error messages? Also, are you sure that you've correctly set up the upstream remote?

Please check if you have the correct URL for the upstream remote and ensure that your login credentials (username and password) are also correct. Double-checking these details might help resolve the issue.

**Comment by saqer23** [comment]
Hi dear

I used react with vite , when I try demo to run data grid with export to
pdf the icons never show up can you tell why and how can I fixed

**Reply by engrmh** [reply]
It seems like you're experiencing an issue with the icons not showing up when using React with Vite and attempting to export data grid to PDF. Here are a few steps you can try to troubleshoot and fix the problem:
1- Verify if the required icon library or font is properly imported and registered in your application. Icons are often provided through external libraries like Font Awesome or Material Icons. Make sure you've followed the installation instructions correctly.

2- Check if the icon component or class name you're using is accurate and matches the specific library you're using. Some libraries have different syntax or naming conventions for accessing icons.

3- Ensure that the CSS for icons is properly included and loaded by your application. Confirm that the CSS file or module responsible for rendering the icons is correctly imported or included in your project's build configuration.

4- If you've recently updated the icon library or your dependencies, there could be compatibility issues. Make sure you're using the right version of the icon library that is compatible with your current React and Vite versions.

5- It's also possible that there might be conflicting CSS styles or customization in your application that affect the rendering of the icons. Inspect your application's styles and ensure that there are no overriding styles or conflicting CSS rules affecting the icons' display.

6- Consider checking the browser console for any error messages related to missing or failed icon requests. This can help identify any specific issues with loading the icons.

7- If the above steps don't resolve the problem, you may want to seek help from the specific data grid library's documentation or community support. They might have encountered similar issues or provide specific guidance for utilizing icons in the PDF export feature.

Remember, the exact solution may depend on the specific libraries, versions, and configurations you're using. Providing more context or error messages related to the issue might help pinpoint the problem and provide a more precise solution.

**Comment by saqer23** [comment]
But same code working when I used js file but when I used jsx it is not
working

**Comment by engrmh** [comment]
Sorry I don’t know 
Maybe not working correctly or not supported in jsx
try to use react-pdf library
